tests/test_fieldgui.py

```diff
@@ -0,0 +1,41 @@
+from PyQt5.QtWidgets import QApplication
```

Comment:
I do not see any testing framework being used in this class - what is its purpose ?